README.md

```diff
@@ -1,14 +1,45 @@
 # Documentation template for OS2 projects
 
-## 🏗️ Getting started 
+## How to use this template for documenting your OS2 project
 
-This is a *bare-minimum* template to create a [Jekyll][Jekyll] site that:
+## 👷 Maintainers guide
 
-- uses the [Just the Docs][Just the Docs] theme;
-- can be built and published on [GitHub Pages][GitHub Pages];
-- can be built and previewed locally, and published on other platforms.
+*In OS2 projects the maintainer team is the group of individuals assigned to oversee the development, quality, and community engagement of the documentation project.*
 
-[Browse documentation][Just the Docs] to learn more about how to use this theme.
+# Setup
+
+1. Start by [creating a documentation repository in your project, from this template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template), including only the main branch.
+3. Configure a [publishing source for GitHub Pages](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages). The documentation site is now live!
+4. Edit and create `.md` [Markdown files](https://guides.github.com/features/mastering-markdown/) to add more content pages to your docs.
```

Comment:
For this to be clear the template must contain the bare minimum of .md files to be filled in. Else my fear is that no additional files are created.

Reply:
I suggest we define a minimum set of files (e.g. "how_to_deploy") and reference these files to Foundation for Public code governance.